libs/ui/label/label.stories.ts

```diff
@@ -50,10 +53,8 @@ export const InputRequired: Story = {
 	render: ({ ...args }) => ({
 		props: { ...args, value: '' },
 		template: `
-    <label hlmLabel>E-Mail *
-        <input [(ngModel)]="value" class='w-80' hlmInput ${argsToTemplate(
-					args,
-				)} type='email' placeholder='Email *' required/>
+    <label hlmLabel ${argsToTemplate(args)}>E-Mail *
+        <input [(ngModel)]="value" class='w-80' hlmInput  type='email' placeholder='Email *' required/>
     </label>
     `,
```

Comment:
Minor: I think you may not have the latest Prettier version in your local node modules. Or maybe the Prettier doesn't recognize stories as angular files?

(Arrow for tab, dot for space)
![image](https://github.com/goetzrobin/spartan/assets/19947758/2fa7621e-695b-4448-8be3-a225a8dc641a)

Reply:
I believe it's as you said about Prettier not recognizing angular in stories.ts. But I could be wrong, someone else can correct me and I can check.

Reply:
@thatsamsonkid seems like it... not sure if its a config issue or limitation...